gwpy/timeseries/timeseries.py

```diff
@@ -1626,6 +1626,12 @@ def gate(self, tzero=1.0, tpad=0.5, whiten=True,
         >>> ax.legend()
         >>> overlay.show()
         """
+        try:
+             from scipy.signal import find_peaks
+        except ImportError:
+             raise ImportError("Must have scipy>=1.1.0 to utilize "
```

Comment:
@tjma12, can you implement it a little different, to preserve the traceback:

```python
        except ImportError as exc:
            exc.args = ("scipy.signal.find_peaks is only available in scipy >= 1.1.0",)
            raise
````

Reply:
Both have been implemented